### PR TITLE
switch from acra to sentry for crash reporting

### PR DESCRIPTION
**Pull request by maks**

### CODE REVIEW

File: app/src/main/java/me/sheimi/sgit/activities/BranchChooserActivity.java [hunk 1]
```diff
@@ -63,6 +63,8 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
 
                 return true;
             case R.id.action_mode_delete:
+                if (true)
+                    throw new RuntimeException("Crasher test.");
```
Comment (cfoote) [inline app/src/main/java/me/sheimi/sgit/activities/BranchChooserActivity.java]: Looks like these lines were for testing and shouldn't have been committed.

File: .travis.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-27.0.3
-    - android-27
+    - build-tools-20.0.3
```
Comment (cfoote) [inline .travis.yml]: Should this be 29.0.3?

Comment (maks) [reply .travis.yml]: ah yes typo well spotted

File: app/src/main/java/me/sheimi/sgit/activities/BranchChooserActivity.java (hunk 1 above)
Comment (maks) [reply app/src/main/java/me/sheimi/sgit/activities/BranchChooserActivity.java]: yep I kept testing while waiting on the CI builds, but I should remove that now

File: .travis.yml (hunk 2 above)
Comment (maks) [reply .travis.yml]: @cfoote thanks for the impromptu review. :+1:

**Pull request merged**